### ISSUE TITLE
Upcoming Dividends Broken Again

### ISSUE THREAD

**Issue by cgailey** [iexg/IEX-API#1403]
It looks like the issue in #1392 is reoccurring.  I haven't seen any announced dividends in the last couple of days.

Thanks.

**Comment by McQgit** [comment]
@cgailey I have been seeing the normal amount of /upcoming-dividends - I look at it every morning.  In fact, I see more than 3900 with this right now at 4:04pm eastern:

https://cloud.iexapis.com/v1/stock/market/upcoming-dividends?token=X&format=csv

**Comment by cgailey** [comment]
Hi @McQgit, while there may be some present, there are many missing. In fact it looks like those in your query for Feb 11 are all OTC listed.

Example missing declared Feb 11:
    The Clorox Company (NYSE: CLX)
    Sterling Bancorp, Inc. (NASDAQ: SBT)
    Terex Corporation (NYSE:TEX)
    Cardinal Health (NYSE: CAH)
    Aberdeen Global Dynamic Dividend Fund (NYSE: AGD)
    KLA Corporation (NASDAQ: KLAC)
    Insteel Industries, Inc. (NasdaqGS: IIIN)
    Kimball International, Inc. (NASDAQ: KBAL)

**Comment by McQgit** [comment]
@cgailey Well, something is broken now as my link above now returns an error message about missing a required 'symbols' parameter - ack!!

That said, it's not unusual for a day lag.  A quick check of my logs showed I saw these (and more) from Feb10 and none are OTC: EXP, JKHY, BLKB, MCY, PEP

Do you have specific examples of earlier than Feb11 that are missing?

**Comment by cgailey** [comment]
@McQgit Feb 10 seems okay now, but I'm sure it was missing many yesterday.  Normally I've seen them in the endpoint the next day, but perhaps the lag has increased?

**Comment by McQgit** [comment]
@cgailey Not sure about lag.  I checked (just) PEP from the above Feb10 items and I saw it on IEX /upcoming about 6am Tue Feb11.  I do see longer lags occasionally.  For me that involves seeing it at an extra day's lag at my 6am check (so, +2 days from announce).  Their docs say they update throughout the day so maybe not as long for them.

As of 6pm Feb12, the only Feb11 announcements are 4 foreign issuers.  If the ones you noted (and others, like HAS) don't show by morning that would be odd. 

FWIW: I'm happy that my sample link above is working again - some wierd transient I suppose or perhaps a typo on my part.
